### PR TITLE
Allow to control emails prefix and signature with config

### PR DESCRIPTION
**Pull request by yohanboniface**
Relates to #912 

Thanks for reviewing :)

### CODE REVIEW

**Comment by Nutomic** [comment]
It would probably make more sense to allow custom email templates. If you go down this path of single variables for elements in the emails, you will end up with dozens of variables. That will be completely unmaintainable.

**Comment by yohanboniface** [comment]
I'd agree in general, but those are two elements that are used in each and every email, so imho it makes sense to have them in a DRY configuration place anyway.

Plus, allowing templates is another story than this one, here we have something done that allows more branded emails, vs something that is to be done, and it's not that simple.

**Comment by Chocobozzz** [comment]
I agree with both opinions. Having email templating would be interesting, but it's not incompatible with signature and object prefix variables.

File: config/default.yaml [hunk 1]
```diff
@@ -53,6 +53,10 @@ smtp:
   ca_file: null # Used for self signed certificates
   from_address: 'admin@example.com'
 
+emails:
+  signature: "PeerTube"
```
Comment (Chocobozzz) [inline config/default.yaml]: Maybe

```
body:
  signature:
```

File: config/default.yaml [hunk 2]
```diff
@@ -53,6 +53,10 @@ smtp:
...
+  prefix: "[PeerTube]"
```
Comment (Chocobozzz) [inline config/default.yaml]: And 

```
object:
  prefix: 
```

File: config/default.yaml [hunk 3]
```diff
@@ -53,6 +53,10 @@ smtp:
   ca_file: null # Used for self signed certificates
   from_address: 'admin@example.com'
 
+emails:
```
Comment (Chocobozzz) [inline config/default.yaml]: `email` for consistency

**Comment by Chocobozzz** [comment]
Thanks @yohanboniface

**Pull request merged**